src/sentry/static/sentry/app/actionCreators/organization.tsx

```diff
@@ -10,15 +10,23 @@ import {Client} from 'app/api';
 import ProjectsStore from 'app/stores/projectsStore';
 import TeamStore from 'app/stores/teamStore';
 import {Organization, Project, Team} from 'app/types';
+import {getPreloadedDataPromise} from 'app/utils/getPreloadedData';
 
 async function fetchOrg(
   api: Client,
   slug: string,
-  detailed: boolean
+  detailed: boolean,
+  isInitialFetch?: boolean
 ): Promise<Organization> {
-  const org = await api.requestPromise(`/organizations/${slug}/`, {
-    query: {detailed: detailed ? 1 : 0},
-  });
+  const org = await getPreloadedDataPromise(
+    'organization',
+    slug,
+    () =>
+      api.requestPromise(`/organizations/${slug}/`, {
```

Comment:
It'll be a good idea to leave comments referencing `src/sentry/static/sentry/index.ejs` file when these URLs/parameters are changed in the future. So that they're sync'd.

Reply:
👍